DESCRIPTION

```diff
@@ -20,6 +20,7 @@ Imports:
     httr2,
     jsonlite,
     magrittr,
+    purrr,
```

Comment:
Assuming we can drop purrr as a dependency now?

Reply:
Yeah actually even the package checks are picking up on it as a note as we don't need it now 
![image](https://github.com/user-attachments/assets/19b0d53e-37ef-4823-a25c-25d37eee45fc)